### Engine ###


# Distances and positions
engine_rate: 500
foot_rise: 0.14
foot_distance: 0.16
kick_windup_distance: 0.25
trunk_height: 0.4
trunk_roll: deg(-3)
trunk_pitch: deg(-29)
trunk_yaw: deg(35.8)

# Timings
move_trunk_time: 0.88
raise_foot_time: 0.75
move_to_ball_time: 0.49
kick_time: 0.11
move_back_time: 0.29
lower_foot_time: 0.06
move_trunk_back_time: 0.36

# Decisions
choose_foot_corridor_width: 0.4

### Stabilizer ###
use_center_of_pressure: false
stabilizing_point_x: -0.04
stabilizing_point_y: -0.01

# these values are used only if use_center_of_pressure is true
pid_x/p: 0.2
pid_x/i: 0
pid_x/d: 0
pid_y/p: 0.5
pid_y/i: 0.0015
pid_y/d: 1.5

### Visualization ###
spline_smoothness: 100
